packages/x-scheduler/src/primitives/use-event-calendar/tests/core.EventCalendarInstance.test.ts

```diff
@@ -0,0 +1,250 @@
+import { getAdapter } from './../../utils/adapter/getAdapter';
+import {
+  DEFAULT_PREFERENCES,
+  DEFAULT_PREFERENCES_MENU_CONFIG,
+  DEFAULT_VIEW,
+  DEFAULT_VIEWS,
+  EventCalendarInstance,
+} from '../EventCalendarInstance';
+import { buildEvent, getIds } from './utils';
+import { CalendarView } from '../../models';
+
+const adapter = getAdapter();
+const DEFAULT_PARAMS = { events: [] };
+
+describe('Core - EventCalendarInstance', () => {
+  describe('create', () => {
+    it('initializes default state', () => {
+      const { store } = EventCalendarInstance.create(DEFAULT_PARAMS, adapter);
+
+      expect(store.state.view).to.equal(DEFAULT_VIEW);
```

Comment:
Nitpick, using `to.deep.equal` could be easier here.
I don't think we want to test the reference equality for the arrays / objects, so loosing that part is fine.